dvc/dagascii.py

```diff
@@ -12,6 +15,42 @@
 from grandalf.routing import EdgeViewer
 from grandalf.routing import route_with_lines
 
+from dvc.env import DVC_PAGER
+
+
+logger = logging.getLogger(__name__)
+
+
+DEFAULT_PAGER = "less"
+DEFAULT_PAGER_FORMATTED = "{} --chop-long-lines --clear-screen".format(
+    DEFAULT_PAGER
+)
+
+
+def find_pager():
+    def make_pager(cmd):
+        def pager(text):
+            return pydoc.tempfilepager(pydoc.plain(text), cmd)
+
+        pager.cmd = cmd
```

Comment:
are you ok with it? or maybe you have a better idea how to test it?

an alternative could be a call + mock, like
```
    with mock.patch.object(os, "system") as m:
        m.return_value = 0
        pager = dagascii.find_pager()
   pager('some-text')
   # pseudo code
   mocked_pydoc_tempfilepager.called_once_with(any, expected_cmd)
```

Reply:
Actually, making `make_pager` a module level fn. would simplify mock and we could check that `make_pager` is called with a given `cmd`..

Reply:
Yeah, a bit hesitant to add this property only because of the test, but I don't have a strong opinion about it. I like your alternative test a bit better since it is more direct.

Reply:
@xliiv considering https://github.com/iterative/dvc/pull/2815#issuecomment-562730313 and the fact that in the current implementation you are testing "cmd = cmd" more than the "make_pager" itself, it seems like `mocked_pydoc_tempfilepager.called_once_with(any, expected_cmd)` would be indeed better.

Reply:
@xliiv ah, but your idea with making `make_pager` a module-level fn is fine too 🙂

Reply:
used a module-level.. seems simpler